ios/FluentUI/Core/Colors.swift

```diff
@@ -275,6 +275,22 @@ public final class Colors: NSObject {
 
     @objc public static func setProvider(provider: ColorProviding, for window: UIWindow) {
         colorProvidersMap.setObject(provider, forKey: window)
+
+        // Reflect the ColorProviding implementation in the FluentUI Vnext stylesheet override for this window
```

Comment:
/// style?

Reply:
You mean "style" instead of "stylesheet"?

Reply:
Yes sorry I mean Xcode commenting style `///`